charter.html

```diff
@@ -0,0 +1,330 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>
+      Anti-Fraud Community Group Charter
+    </title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <link rel="stylesheet" href="//www.w3.org/StyleSheets/TR/base">
+    <style>
+      body {
+        max-width: 60em;
+        margin: auto;;
+      }
+      *:target {
+        background-color: yellow;
+      }
+      li {
+        margin-bottom: 9pt;
+      }
+      .note {
+        background-color: yellow;
+        padding: 10px;
+      }
+      .remove {
+        background-color: yellow;
+      }
+    </style>
+  </head>
+  <body>
+    <h1>
+      [DRAFT] Anti-Fraud Community Group Charter
+    </h1>
+    <p>
+      This Charter is work in progress. To submit feedback, please use <a href="https://github.com/antifraudcg/antifraudcg.github.io/issues">https://github.com/antifraudcg/antifraudcg.github.io/issues</a>.
+    </p>
+    <h2 id="goals">
+      Goals
+    </h2>
+    <p>
+      The mission of the Anti Fraud Community Group is to identify and define
+      scenarios involving fraud and unwanted traffic, as well as to incubate
+      and develop web features and APIs to address those scenarios while being
+      mindful of user security, privacy, and accessibility. Fraud and unwanted
+      traffic can include web activity perpetrated by botnets,attackers
+      impersonating users, and other activity that intends to deceive and harm
+      users or compromise web services.
+    </p>
+    <p>
+      The group welcomes participation from anti-fraud service providers,
+      common targets of unwanted traffic, browser vendors, web privacy
+      advocates, web application developers, web hosting and cloud service
+      providers, and other interested parties.
+    </p>
+    <p>
+      The Community Group will discuss issues that server operators (network
+      and web server admins) and anti-fraud service providers face in this
+      area, as well as ideas for new web features and APIs intended to be
+      implemented in browsers or similar user agents.
```

Comment:
I'm not sure we want to limit the scope to implementation in browsers and user agents. Many web features include implementation on the server side as well, or could involve servers/issuers/relying parties etc. 

The out-of-scope section directly notes that features that don't involve the client at all are out-of-scope (+1).

Reply:
If members identify means to advance our charter's goals via a design pattern (or protocol) that leverages existing browser components, should this proposal not be in scope? Constraining the solution set to ideas that require browser changes seems limiting in this regard. Broadening this to "as well as ideas for new web features, protocols, and APIs intended to be implemented in browsers or similar user agents" would accommodate these proposals.